OpenEdXMobile/src/main/java/org/edx/mobile/http/callback/ErrorHandlingOkCallback.java

```diff
@@ -86,29 +96,31 @@ public ErrorHandlingOkCallback(@NonNull final Context context,
      *                         thus invokes that start callback immediately as well.
      */
     public ErrorHandlingOkCallback(@NonNull final Context context,
+                                   @NonNull final Class<T> responseBodyClass,
                                    @NonNull final CallTrigger callTrigger,
                                    @Nullable final TaskProgressCallback progressCallback) {
-        this(context, callTrigger,
+        this(context, responseBodyClass, callTrigger,
                 progressCallback,
                 context instanceof TaskMessageCallback ? (TaskMessageCallback) context : null);
     }
 
     /**
      * Create a new instance of this class.
-     *
```

Comment:
Yup, and seems like instead of a carriage return, the next line got an extra space @ start 😛